bindgen-tests/tests/parse_callbacks/mod.rs

```diff
@@ -86,25 +87,63 @@ impl ParseCallbacks for BlocklistedTypeImplementsTrait {
     }
 }
 
+#[derive(Debug)]
+struct FieldVisibility {
```

Comment:
could you document how this callback works so other contributors don't have to read the `ParseCallbacks` implementation if they were to change the tests?